cmd/rookd/daemon.go

```diff
@@ -47,7 +47,7 @@ func runDaemon(cmd *cobra.Command, args []string) error {
 	if err := flags.VerifyRequiredFlags(daemonCmd, []string{"type"}); err != nil {
 		return err
 	}
-	if daemonType != "mon" && daemonType != "osd" && daemonType != "mds" && daemonType != "rgw" {
+	if daemonType != "mon" && daemonType != "osd" && daemonType != "mds" && daemonType != "rgw" && daemonType != "rgw-admin" {
 		return fmt.Errorf("unknown daemon type: %s", daemonType)
```

Comment:
the tool writes its output to stdout so the only way to capture that is to launch a new process, right?